src/pages/VenueLandingPage/VenueLandingPageContent.tsx

```diff
@@ -36,20 +37,20 @@ dayjs.extend(advancedFormat);
 
 type VenueLandingPageContentProps = {
   venue: WithId<AnyVenue>;
+  world: WithId<World>;
   withJoinEvent?: boolean;
 };
 const VenueLandingPageContent: React.FC<VenueLandingPageContentProps> = ({
   venue,
+  world,
   withJoinEvent = true,
 }) => {
   const venueEvents = useSelector(venueEventsSelector);
 
   const spaceSlug = venue.slug;
 
-  const { world } = useWorldById(venue?.worldId);
-
   const [validBannerImageUrl] = useValidImage(
-    venue?.config?.landingPageConfig.bannerImageUrl,
+    world?.config?.landingPageConfig.coverImageUrl,
```

Comment:
I think this needs to be a fallback.
So, we try the space landing cover if there is one. Otherwise, we try the world cover image.

There is another issue that @alexey-petrov is fixing for those settings not being configurable at the moment!
https://github.com/sparkletown/internal-sparkle-issues/issues/1562

Reply:
Got it!